### PR TITLE
RN-CNV-7548 Release note known issue

### PR DESCRIPTION
**Pull request by sjhala-ccs**
Known issue for 2.5 release notes: https://issues.redhat.com/browse/CNV-7548

SME review requested from Stu Gott (tagged in Jira)
QE review requested from Vasiliy Sibirskiy (tagged in Jira)

### CODE REVIEW

**Review by stu-gott — DISMISSED** [review]
LGTM

**Review by vsibirsk — DISMISSED** [review]
lgtm

File: virt/virt-2-5-release-notes.adoc [hunk 1]
```diff
@@ -112,7 +112,11 @@ The *Pending Changes* banner at the top of the page displays a list of all chang
 
 //This section is work in progress
 
-//This defect is ON_QA
+* When a node fails in user-provisioned installations of {product-title} on bare metal deployments, the virtual machine does not automatically restart on another node. Auto restart is supported only for installer-provisioned installations that have machine health checks enabled. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886437[*BZ#1886437*])
```
Comment (bergerhoffer) [inline virt/virt-2-5-release-notes.adoc:115]: Should "Auto" be "Automatic"?

**Review by bergerhoffer — COMMENTED** [review]
Just one comment.

**Comment by sjhala-ccs** [comment]
@bergerhoffer This is ready for you to merge. Thanks!

**Comment by sjhala-ccs** [comment]
@bergerhoffer This is ready for you to merge. Please note that when I was making the suggested changes for this PR, I realized that I had forgotten to add bullets (*) for two other release note stories that are already merged. I have added those changes to this PR as well as removed two resolved known issues from 2.4. 
Sorry for the additional changes.

**Comment by bergerhoffer** [comment]
No worries, LGTM. Merging!

**Pull request merged**